features/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -23,6 +23,13 @@
         {% endfor %}
         {% endif %}
     },
+    "analytics": {
+        {% if console.analytics.metrics is defined %}
+        {% for param, value in console.analytics.metrics.items() %}
```

Comment:
If `console.analytics.metrics is defined` we need to add `metricts` object right?
```suggestion
       "metricts" {
        {% for param, value in console.analytics.metrics.items() %}
```

Reply:
fixed with 52d1c5b